src/libs/OptionsListUtils.js

```diff
@@ -204,20 +205,58 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report) && !_.contains(participants, report.ownerEmail)) {
+        participants.push(report.ownerEmail);
+    }
+    return participants;
+}
+
+/**
+ * Get the Avatar urls or return the icon according to the chat type
+ *
+ * @param {Object} report
+ * @returns {Array<*>}
+ */
+function getAvatarSources(report) {
+    return _.map(lodashGet(report, 'icons', ['']), (source) => {
```

Comment:
If can, please add some kind of comment here about why we are defaulting to an array with an empty string.